swaap/swaap/View Controllers/RootTabBarController.swift

```diff
@@ -10,25 +10,18 @@ import UIKit
 
 class RootTabBarController: UITabBarController {
 	let authManager = AuthManager()
-	// top level coordinators go here - they will be passed in as arguments to the initializer
-	var authCoordinator: AuthCoordinator?
-	lazy var contactsCoordinator = ContactsCoordinator(contactsController: self.contactsController)
 
 	/// property observer (cannot present a view when its parent isn't part of the view hierarchy, so we need to watch
 	/// for when the parent is in the hierarchy
 	private var windowObserver: NSKeyValueObservation?
 	private var credentialObserver: NSObjectProtocol?
 
 	let contactsController = ContactsController()
+	var rootAuthVC: RootAuthViewController?
```

Comment:
this should actually probably be refactored to be a lazy initializer

would prevent the possibility of showing multiple stacked rootAuthVCs which would in turn clean up some of the logic in `runAuthCoordinator`